docs/ref/utils.txt

```diff
@@ -856,6 +861,10 @@ appropriate entities.
     Can be called multiple times on a single string (the resulting escaping is
     only applied once).
 
+    .. versionchanged:: 1.11
```

Comment:
As [noted on the ticket](https://code.djangoproject.com/ticket/10107#comment:12), `mark_for_escaping()` is deprecated, so there's no use in making any changes to it. That should fix the build failure.